component/main.jsonnet

```diff
@@ -30,7 +30,44 @@ local opaqueSecrets() = [
   for name in std.objectFields(params.opaque)
 ];
 
+local tlsSecrets() = [
+  local values = std.get(params.tls, name);
+  {
+    apiVersion: 'v1',
+    kind: 'Secret',
+    metadata: {
+      name: namespacedName(name, '').name,
+      namespace: namespacedName(name, '').namespace,
+    },
+    type: 'kubernetes.io/tls',
+    data: {
```

Comment:
That won't work. You need to write the data to `stringData` as it's not base64 encoded

Reply:
Fixed